server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/SignificantLongTermsAggregator.java

```diff
@@ -47,7 +48,7 @@ public SignificantLongTermsAggregator(String name, AggregatorFactories factories
             IncludeExclude.LongFilter includeExclude, Map<String, Object> metadata) throws IOException {
 
         super(name, factories, valuesSource, format, null, bucketCountThresholds, context, parent,
-                SubAggCollectionMode.BREADTH_FIRST, false, includeExclude, false, metadata);
+                SubAggCollectionMode.BREADTH_FIRST, false, includeExclude, CardinalityUpperBound.ONE, metadata);
```

Comment:
Why is this always ONE?

Reply:
This has since been removed. It was `ONE` because that agg was wrapped in `asMultiBucketAggregator`.